week-5-ci-and-production/w05e02-continuous-deployment/.github/workflows/api.yml

```diff
@@ -0,0 +1,38 @@
+name: API – Stage Deploy
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'shopping-lister-api/**'
+
+jobs:
+  test_build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Cache node modules
+        uses: actions/cache@v3
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: Install dependencies
+        run: npm ci
+        working-directory: ./shopping-lister-api
+      - name: Format code and check for errors
+        run: npm run lint:full
+        working-directory: ./shopping-lister-api
+      - name: Run tests
+        run: npm run test
+        working-directory: ./shopping-lister-api
+      - name: Build project
+        run: npm run build
+        working-directory: ./shopping-lister-api
+      - name: Deploy
+        run: |
+          mkdir -p ~/shopping-lister-api-stage
```

Comment:
zasadniczo chyba `cp` zrobi katalog - nie musisz robić kroku `mkdir`

Reply:
Faktycznie tak to działa 👍 Poprawiłem